h/static/scripts/group-forms/utils/api.ts

```diff
@@ -113,10 +113,10 @@ export type APIOptions = {
   signal?: AbortSignal;
 
   /** Index of first item to return in paginated APIs. */
```

Comment:
This comment needs to be updated to eg. "1-based number of first page to return ..."